frontend/tests/features/collection/revision.test.ts

```diff
@@ -194,7 +194,7 @@ async function startRevision(page: Page): Promise<string> {
    * `COLLECTION_ROW_ID` selector element
    * @see https://playwright.dev/docs/selectors#intermediate-matches
    */
-  const COLLECTION_ROW_SELECTOR_START = `*${getTestID(
+  const COLLECTION_ROW_SELECTOR_START = `*${page.getByTestId(
```

Comment:
Hmm not sure if this works, since `page.getByTestId` returns a Locator object, so I my guess is that turning locator into a string would be `[object Object]` ? (I haven't testesd this locally 😆 )

Reply:
Yeah, this is mistake - will revert. The tests are skipped locally hence did not catch